test/framework/kogitoapp.go

```diff
@@ -19,8 +19,10 @@ import (
 	"strings"
```

Comment:
I guess this file shouldn't be here :laughing: . Maybe `git pull --rebase upstream master` before squashing? Try to revert via `git reflog`.

Reply:
Oops, I had done that, but _before_ I started working on this. D'oh. Then I didn't realize it when submitting the PR. D'oh².

Hehe I'll fix this in a bit, thanks!